api/HTMLElement.json

```diff
@@ -2231,6 +2231,46 @@
           }
         }
       },
+      "scroll_event": {
```

Comment:
Can you move this entry to `Element` instead? I'm pretty sure that with the right CSS, this event could also fire on `SVGElement` or some other type of `Element`.